factcast-store/src/main/java/org/factcast/store/internal/PgFactStream.java

```diff
@@ -74,7 +74,7 @@ public class PgFactStream {
 
   void connect(@NonNull SubscriptionRequestTO request) {
     this.request = request;
-    log.debug("{} connect subscription {}", request, request.dump());
```

Comment:
is that really intended? 
req.dump() was supposed to be useful for debug purposes, where req.toString() is only the name.